flink-end-to-end-tests/test-scripts/common_ha.sh

```diff
@@ -49,7 +49,7 @@ function verify_num_occurences_in_logs() {
     local text="$2"
     local expected_no="$3"
 
-    local actual_no=$(grep -r --include "*${log_pattern}*.log" -e "${text}" "$FLINK_LOG_DIR/" | cut -d ":" -f 1 | uniq | wc -l)
+    local actual_no=$(grep -r --include "*${log_pattern}*.log*" -e "${text}" "$FLINK_LOG_DIR/" | cut -d ":" -f 1 | sed "s/\.log\.[0-9]\{1,\}$/.log/g" | uniq | wc -l)
```

Comment:
```suggestion
    local actual_no=$(grep -r --include "*${log_pattern}*.log*" -e "${text}" "$FLINK_LOG_DIR/" | cut -d ":" -f 1 | sed "s/log[.0-9]*$//g" | uniq | wc -l)
```
we can simplify this a bit

Reply:
You're right. Initially, my intention was to be more strict on which file names to adjust. But that's not necessary due to the preceding `grep` call which only allows `*.log*` files to be included